src/main/java/com/gal/trivia/entity/Answer.java

```diff
@@ -1,28 +1,25 @@
 package com.gal.trivia.entity;
 
-import lombok.Data;
+import lombok.*;
 
 import javax.persistence.*;
 
 @Entity
-@Data
+@Getter
```

Comment:
In place of Getter, Setter AllArgsConstructor we can use @Data which supports all 3 together.